### ISSUE TITLE
Right Click function

### ISSUE THREAD

**Issue by beastyrabbit** [fhchl/languagetool#3]
Hello, 
If i want to "show suggestions"
first have to click into the highlighted Area to and then right click. 

It would be much better if i don't have to click there first. Just mouse-over it.

**Comment by fhchl** [comment]
Good idea! Would you like to help implementing that feature? I am currently quite busy with my PhD ... :)

**Comment by beastyrabbit** [comment]
I'm also writing my Bachlorthesis. So no time at the moment :)
If i don't forget i will look into it but it will take a few month.

**Comment by beastyrabbit** [comment]
I found this:
https://github.com/facebooknuclide/hyperclick
I think that can solve the right click problematic.
